plugin/storage/es/mappings/jaeger-span.json

```diff
@@ -1,5 +1,8 @@
 {
-  "template": "*jaeger-span-*",
+  "template": [
+    "*jaeger-span-*",
+    "-jaeger-span-archive"
```

Comment:
1. why does this not have `*`s at both ends?
2. if this template already covers archive, why do you need `jaeger-span-archive.json`?

Reply:
What I meant was: why can't we parameterize this file and reuse it for both, instead of creating a nearly identical duplicate? E.g.
```
  "template": [
    "{{ .Template1 }}",
    "{{ .Template2 }}"
```

Reply:
> 

Is it possible to run the integration tests on local? I don't seem to be able to get past the health check stage of `elasticsearch_test.go`

Reply:
I think you should be able to run the test with `bash scripts/es-integration-test.sh elasticsearch 7.14.0`